TypeCobol/Compiler/Diagnostics/CrossChecker.cs

```diff
@@ -726,6 +726,21 @@ private static SymbolReference ResolveProcedureName(SymbolTable table, SymbolRef
         protected static void Check<T>(T node, [NotNull] IList<T> found) where T : Node
         {
             if (found.Count > 1) DiagnosticUtils.AddError(node, "Symbol \'" + node.Name + "\' already declared");
+
+            // in a section/paragraph, "STOP RUN" or "GOBACK" should be the last statement
+            // search if paragraph
+            bool statementAfterFound = false;
```

Comment:
It works, but it's not optimized.
A standard Cobol program will have more paragraph than Goback/stop run statement.

It'll be better to start this check on goback and stop run statements (the main visitor of CrossChecker already visits these elements).

Reply:
done
https://github.com/TypeCobolTeam/TypeCobol/tree/b9c4302cf153f2afae9c1fcc2e5c314d9df95a08